pr2eus/robot-interface.l

```diff
@@ -44,6 +45,50 @@
   (:time-to-finish ()
     (ros::ros-debug "[~A] time-to-finish ~A" ros::name-space time-to-finish)
     time-to-finish)
+  (:action-result-cb (msg)
+    "
+    display error messages from [depending on robots]/follow_joint_trajectory/result
+    "
+    ;; The following error messages will not appear for the first (send *ri* :angle-vector)
+    ;; To display it, use :wait-interpolation or (ros::spin-once groupname)
+    (let ((status-num (send (send msg :status) :status))
+          (status-text (send (send msg :status) :text))
+          (error-code nil)
+          (error-word '(SUCCESSFUL
+                        INVALID_GOAL
+                        INVALID_JOINTS
+                        OLD_HEADER_TIMESTAMP
+                        PATH_TOLERANCE_VIOLATED
+                        GOAL_TOLERANCE_VIOLATED)))
```

Comment:
Can we make these strings to avoid making new symbols?

Reply:
Yes, but since there has been no mapping between these strings and `error_code` yet, we will have to send a pull request to roseus or something.

Reply:
Maybe I was unclear. I mean:
```lisp
(let ((error-word '("SUCCESSFUL"
                    "INVALID_GOAL"
                    "INVALID_JOINTS"
                    "OLD_HEADER_TIMESTAMP"
                    "PATH_TOLERANCE_VIOLATED"
                    "GOAL_TOLERANCE_VIOLATED")))
  ...)
```

Reply:
Sorry. I see.
I updated in [2ce0b4c](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/460/commits/2ce0b4c25bdaa5d9e91c4e6bc4bd4c2f84f06a13)